python/pyspark/errors/errors.py

```diff
@@ -0,0 +1,93 @@
+#
+# Licensed to the Apache Software Foundation (ASF) under one or more
+# contributor license agreements.  See the NOTICE file distributed with
+# this work for additional information regarding copyright ownership.
+# The ASF licenses this file to You under the Apache License, Version 2.0
+# (the "License"); you may not use this file except in compliance with
+# the License.  You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+from typing import Union, Any, Type
+from pyspark.errors.exceptions import PySparkException
+from py4j.java_gateway import JavaClass
+
+
+def _get_pyspark_errors() -> JavaClass:
+    from pyspark.sql import SparkSession
+
+    spark = SparkSession._getActiveSessionOrCreate()
+    assert spark._jvm is not None
+    return spark._jvm.org.apache.spark.python.errors.PySparkErrors
+
+
+def column_in_list_error(func_name: str) -> "PySparkException":
+    pyspark_errors = _get_pyspark_errors()
+    e = pyspark_errors.columnInListError(func_name)
```

Comment:
Invoking a function on a Scala object just to access a JSON file feels wrong to me.